### ISSUE TITLE
OpenGL issue crashes MyPal

### ISSUE THREAD

**Issue by BossSmiley** [Feodor2/Mypal68#522]
I've been using the MyPal browsers for over a year now, and never had the following problem : lately, every time I use a MyPal browser (the last 3 versions), it ultimately crashes on me with an error saying, 'GLContext WGL' application error. Then it tells me to shut down the browser or debug it choosing OK or Cancel. My question is, why am I getting these errors, all of a sudden ? This happens, without fail, at some point, every time I use one of the last 3 browser versions. I have not done anything new to my XP SP3 system, so is this a MyPal browser issue or something in my system that is causing this ? Either or, can this be fixed somehow ? Thanks for listening !

**Comment by jonm58** [comment]
OpenGL Context is part of the complete OpenGL implementation. OpenGL will not exist until the OpenGL,Context is created. Created by the APIs of different platforms themselves. The one based on Windows platform is called WGL.

So what exactly does your PC use CPU and GPU?

**Comment by Feodor2** [comment]
Yes the issue without details consider as invalid, try disbale webgl look #376
